app/scripts/models/reliers/oauth.js

```diff
@@ -187,7 +187,7 @@ define([
       var self = this;
       var clientId = self.get('clientId');
 
-      return self._oAuthClient.getClientInfo(clientId)
+      return self._fxaClient.getOAuthClientInfo(clientId)
```

Comment:
Another possible option is to have the oauth broker fetch the info and hand it to the relier. Does that bloat the oauth broker?